backend/handlers/settings.go

```diff
@@ -145,7 +140,7 @@ func AddAccountDetails(postgresDb *gorm.DB) func(w http.ResponseWriter, r *http.
 		}
 
 		// Trigger scan
-		if err := control.TriggerScan(postgresDb); err != nil {
+		if err := control.TriggerScan(postgresDb, ad.AccountName); err != nil {
```

Comment:
do we need TriggerScan here or will ManualAddAccountDetails handle it?

Reply:
yes, it will handle it